### ISSUE TITLE
Can I edit an exported project

### ISSUE THREAD

**Issue by A2000J** [google/marzipano#470]
Hi,
I love the Marzipano tool. It works great with the images from my Insta360 X4.
Is there a way to reload/import an exported project?
Sometimes I would like to make a change afterwards. To do this, I always create the project completely from scratch.
Is there a better way?
Regards Andreas

**Comment by raundalrahul** [comment]
yes!!. you can by changing data.js adding tiles and respective html

**Comment by A2000J** [comment]
What I mean is. How can I use the editor again to change an existing project?

**Comment by neuhaus** [comment]
Dương Tùng Anh has vibe-coded a version of Marzipan Tool that allows importing a previous export and editing! A highly requested missing feature of Marzipan Tool.

He mentioned it at
https://github.com/google/marzipano/issues/309#issuecomment-3626747731
and you can find it hosted at
https://marzipano-importer.tunnaduong.com/
with the source code at 
https://github.com/tunnaduong/marzipano-importer
